qt/python/mantidqt/widgets/plotconfigdialog/__init__.py

```diff
@@ -0,0 +1,104 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+
```

Comment:
Could you check all of the files and make sure they all at least have the `from __future__ import (absolute_import, division, unicode_literals)` imports?